packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -85,7 +85,8 @@ const REFRESH_STRING = '- [x] Refresh this comment';
 // Github issue comment API has a limit of 65536 characters.
 const MAX_CHARS_IN_COMMENT = 64000;
 
-const ALLOWED_ORGANIZATIONS = ['googleapis', 'GoogleCloudPlatform'];
+//tmatsuo is present for testing
```

Comment:
tmatsuo no longer works on the project. Please remove them from the allow-list.

Reply:
done, I had to update all the test data to use an allowlisted repo (before it was tmatsuo)